chessence-frontend/src/pages/Archive/ArchivePage.js

```diff
@@ -5,30 +5,29 @@ import GameList from "./GameList/GameList.js";
 import GameMove from "./Moves/GameMove.js";
 
 export default function ArchivePage() {
-    //const [games, setGames] = useState([]);
-
-    // async function fetchAll(){
-    //     try {
-    //        const response = await axios.get('http://localhost:8000/archive');
-    //        return response.data.game_list;
-    //     }
-    //     catch (error){
-    //        //We're not handling errors. Just logging into the console.
-    //        console.log(error);
-    //        return false;
-    //     }
-    //  }
-
-    //useEffect(() => {
-    //    fetchAll().then( result => {
-    //      if (result)
-    //          setGames(result);
-    //    });
-    //  }, [] );
-
+    const [games, setGames] = useState([]);
     const [showMove, setShowMove] = useState(false);
     const [gameId, setGameId] = useState(0);
 
+    async function fetchAll() {
+        try {
+            const response = await axios.get(
+                "https://chessence.azurewebsites.net/history"
```

Comment:
we should eventually set up a `.env` with a REACT_APP_BACKEND variable instead of hard coding `chessence.azurewebsites.net` or `localhost:8000`.